pkg/scheduler/event_handler.go

```diff
@@ -177,10 +177,19 @@ func (s *Scheduler) enqueueAffectedBindings(oldCluster, newCluster *clusterv1alp
 	bindings, _ := s.bindingLister.List(labels.Everything())
 	for _, binding := range bindings {
 		placementPtr := binding.Spec.Placement
+		// never reach here
 		if placementPtr == nil {
```

Comment:
> // never reach here

This comment would be better placed inside `if`.

Reply:
updated

Reply:
+1